automation/repeated_test.sh

```diff
@@ -31,6 +31,9 @@ set -euo pipefail
 # * run all changed tests five times and
 # * randomize the test order each time
 
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+KUBEVIRT_ROOT=$(readlink --canonicalize ${BASEDIR}/..)
```

Comment:
That `&&` stresses me out... If `cd` fails for some weird reason, $BASEDIR is empty and $KUBEVIRT_ROOT is `/`.
That means we're giving our entire rootfs as `/kubevirt/` to the container below :sweat_smile:

Reply:
ack, will correct - I actually found something simpler, hope you like it better.

being curious: is that a real concern as

Reply:
Indeed, good point! But now we should be safe even if that ever gets removed :)